rust_src/src/keymap.rs

```diff
@@ -773,4 +773,15 @@ pub fn apropos_internal(regexp: LispStringRef, predicate: LispObject) -> LispObj
     }
 }
 
+/// Return a list of keymaps for the minor modes of the current buffer.
+#[lisp_fn]
+pub fn current_minor_mode_maps() -> LispObject {
+    // maps is initialized to null to silence the borrow checker. If this is
+    // not done then it will not compile since maps is not initialized
+    let mut maps: *mut LispObject = ptr::null_mut();
+    unsafe {
+        let num_of_maps = current_minor_maps(ptr::null_mut(), &mut maps);
+        Flist(num_of_maps, maps)
```

Comment:
We usually try not to use 'F' functions in Rust. But I am torn over whether `list([num_of_maps, maps])` is preferable.

Reply:
I'd personally prefer it.

Reply:
Actually, I think this might be better written `list!(num_of_maps, maps)`.